src/lib.rs

```diff
@@ -18,6 +18,9 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
+// Remove this note before publishing
+//! # ⚠️ WARNING You are viewing the master docs
+//!
```

Comment:
Pretty sure we are going to forget about this. I don't think we need to explicitly point this out. I'd expect users who navigate to https://libp2p.github.io/rust-libp2p/libp2p/ to understand that they are viewing the master docs and not of a released version.

Reply:
Ok, will remove it then.